packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -627,17 +627,17 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
      * Load the stores from a snapshot and returns the runtime.
      * @param context - Context of the container.
      * @param registryEntries - Mapping to the stores.
+     * @param existing - When loading from an existing snapshot
      * @param requestHandler - Request handlers for the container runtime
      * @param runtimeOptions - Additional options to be passed to the runtime
-     * @param existing - (optional) When loading from an existing snapshot. Precedes context.existing if provided
      */
     public static async load(
         context: IContainerContext,
         registryEntries: NamedFluidDataStoreRegistryEntries,
+        existing: boolean,
```

Comment:
This is going to be problematic for back-compat reasons which will block the change due to some tests failing. 

Because this was an optional parameter at the end of the list of parameters, unfortunately you can't just make it mandatory, so you're forced to move it at the end of the list of required parameters. So now you have to pay the price for backwards compatibility, as unfortunately new client code supplying the existing parameter may supply it to older runtime instances, which expect a function at the 3rd position instead of a boolean.

We can talk on Monday about strategies to deal with this.